### PR TITLE
Website: Introduce Snippets for FE Package Managers

### PR DESCRIPTION
**Pull request by jackw**
**What this PR does / why we need it**:
This PR adds the ability to load markdown snippets inside syntax highlighted code blocks. Right now it can be used to tab between different package managers.

image

It can also be used in combination with the `SyncCommand` component to update inline code blocks:

https://user-images.githubusercontent.com/73201/235719348-23ab98ae-cccd-421b-9c22-0024e0a2a15f.mp4

**Which issue(s) this PR fixes**:

Related issue: #156

**Special notes for your reviewer**:

### CODE REVIEW

File: docusaurus/website/src/components/CodeSnippets/CodeSnippets.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+function CodeSnippets({ paths, groupId, queryString }) {
+  const snippetPaths = paths.split(',').map((path) => path.trim());
+  const [snippets, setSnippets] = useState([]);
+
+  useEffect(() => {
+    async function fetchSnippets() {
+      const fetchedSnippets = await Promise.all(
+        snippetPaths.map(async (path) => {
+          const [_, value, language] = path.split('.');
```
Comment (leventebalogh) [inline docusaurus/website/src/components/CodeSnippets/CodeSnippets.tsx:14]: I would probably add a block of comments above the component definition that explains how to use this component on the call-site, especially how the name of the file relates to the names of the tabs.

**Review by leventebalogh — APPROVED** [review]
Magic! 🪄🎩 
Nice one, LGTM 👍

**Review by mckn — APPROVED** [review]
LGTM!

**Review by leventebalogh — APPROVED** [review]
LGTM 👍

**Pull request merged**